WoWsShipBuilder.Common/Features/ShipSelection/FittingToolSelector.razor

```diff
@@ -0,0 +1,150 @@
+                                                                    @foreach (var ship in chunk)
+                                                                    {
+                                                                        <div class="ft-selector-ship-name-grid" @key="ship.Index" @onclick="@(_ => SelectShip(ship))">
+                                                                            @if (AppSettings.FittingToolSelectorShowShipImages)
+                                                                            {
+                                                                                <MudImage Src="@($"{CdnOptions.Value.Host}{CdnOptions.Value.ShipImagePath}/{ship.Index}.png")" Height="30" ObjectPosition="ObjectPosition.Center"/>
+                                                                            }
+                                                                            <MudText Class="pa-2" Style="white-space: nowrap; font-size: 1rem">
+                                                                                @Localizer.SimpleGameLocalization(ship.Index)
+                                                                            </MudText>
+                                                                        </div>
+                                                                    }
+                                                                </MudStack>
+                                                            }
+                                                        </MudStack>
+                                                    }
+                                                </MudStack>
+                                                if (tier.Key != shipClass.Value.MaxBy(x => x.Key).Key)
+                                                {
+                                                    <MudDivider Light="false" DividerType="DividerType.FullWidth"/>
+                                                }
+                                            }
+                                        </ul>
+                                    </li>
+                                }
+                            </ul>
+                        </li>
+                    }
+                </ul>
+            </li>
+        }
+    </ul>
+    <MudButton Style="z-index: 900; min-height: 531px; min-width: 20px;" Color="Color.Primary" Variant="Variant.Outlined" Class="px-0" Size="Size.Small" OnClick="@(() => openDrawer = !openDrawer)">
+        @(openDrawer ? "<" : ">")
+    </MudButton>
+</aside>
+}
+
+@code
+{
+    [Parameter]
+    public IList<ShipBuildContainer> SelectedShips { get; set; } = default!;
+
+    private readonly ImmutableDictionary<Nation, ImmutableDictionary<ShipCategory, ImmutableDictionary<ShipClass, ImmutableDictionary<int, ImmutableList<Ship>>>>> dataStructure = AppData.FittingToolShipSelectorDataStructure;
```

Comment:
Why is the value from AppData copied? It's an immutable data structure, just share the one instance from AppData directly.
Also, the name of this is still not clear, dataStructure is not really more expressive than treeStructure